include/core/parameters.h

```diff
@@ -475,6 +480,12 @@ namespace Parameters
       immersed_boundary_tanh
     } tracer_diffusivity_model;
 
+    enum class TracerReactionConstantModel
+    {
+      constant,
+      immersed_boundary_tanh
```

Comment:
I was wondering maybe why not have a "none" model and then you would use taht to decide if you assemble or not the reaction? This would remove the additional bool variable